sql: populate spatial_ref_sys table with projection entries

otan opened the pull request:
Also fixed a bug where `\0` didn't actually output a NULL terminator (no
idea why it worked).

Release note (sql change): Populate the spatial_ref_sys table with
support SRID entries for geospatial data types.

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/49887)
<!-- Reviewable:end -->

otan committed:
sql: populate spatial_ref_sys table with projection entries

Also fixed a bug where `\0` didn't actually output a NULL terminator (no
idea why it worked).

Release note (sql change): Populate the spatial_ref_sys table with
support SRID entries for geospatial data types.

sumeerbhola reviewed (approved):
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/> 

> Reviewed 6 of 6 files at r1.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/49887#-:-M9IpV9qCFcWu4-ul1IQ:bg7igkv)* status: :shipit: complete! 1 of 0 LGTMs obtained



<!-- Sent from Reviewable.io -->

otan commented:
bors r=sumeerbhola

otan commented:
bors r-
